deploy/charts/rook-ceph/templates/role.yaml

```diff
@@ -37,18 +37,6 @@ rules:
   - create
   - update
   - delete
-- apiGroups:
```

Comment:
Don't we need this when `monitoring.enabled: true`?

Reply:
That is old and for the operator chart. The monitoring RBAC is deployed by the cluster chart now.